CODE/Logicytics.py

```
@@ -239,26 +237,30 @@
         log.warning("This flag will use threading!")
 
+    if ACTION == "vulnscan_ai":
+        # Only vulnscan detector
+        execution_list = ["vulnscan.py"]
+
     log.debug(f"The following will be executed: {execution_list}")
     return execution_list
 
 
-@log.function
 def execute_scripts():
```

Ruff (0.8.0)

`execute_scripts` is too complex (13 > 10)

(C901)